### PR TITLE
Start RTD restructuring

### PR DESCRIPTION
**Pull request by johnmhoran**
@majurg This is just the initial stage of the restructuring and fleshing-out work -- more to come.

### CODE REVIEW

File: docs/source/getting-started/index.rst [hunk 1]
```diff
@@ -1,38 +1,66 @@
...
+====================
 
-Import a ScanCode JSON file, and see what components are in your software!
+We've also provided a set of `sample scans <https://github.com/nexB/scancode-workbench/tree/develop/samples>`__
+that you can  review in ScanCode Workbench in order to get a sense of its functionality and the
+types of information captured by a scan.
 
-If you're ready, keep on reading this documentation for more information on how to use
-ScanCode Workbench.
+:guilabel:`[The samples need to be updated from 2.9.2 to the current version of SCTK.]`
```
Comment (steven-esser) [inline docs/source/getting-started/index.rst]: Yes they do ;) Can you enter a ticket for this (if we do not already have one). It doesnt need to be super detailed, just this sentence.

File: docs/source/how-to-guides/load-your-data/open-save-sqlite-file.rst [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
-===================================
+====================================
+:index:`Open and Save a SQLite File`
+====================================
 
 Open a SQLite File
 ==================
 
+Once you've imported your JSON scan and saved it as SQLite, in most cases you'll continue your
+review and analysis work solely in the SQLite file.
```
Comment (steven-esser) [inline docs/source/how-to-guides/load-your-data/open-save-sqlite-file.rst]: I would get rid of "in most cases". Whether the user wants it or not, all the reviewing and analysis work is done via the sqlite file (by design).

File: docs/source/license.rst [hunk 3]
```diff
@@ -1,7 +1,9 @@
...
+.. _license:
+
+================
+:index:`License`
+================
 
 -  Apache-2.0
 -  Multiple licenses (LGPL, MIT, BSD, etc.) for third-party components.
--  See the `NOTICE <https://github.com/nexB/scancode-workbench/blob/develop/NOTICE>`__ file for more details.  *[The copyright date needs to be updated or removed.]*
+-  See the `NOTICE <https://github.com/nexB/scancode-workbench/blob/develop/NOTICE>`__ file for more details.  :guilabel:`[The copyright date needs to be updated or removed.]`
```
Comment (steven-esser) [inline docs/source/license.rst]: We could use a ticket for this task as well.

**Review by steven-esser — COMMENTED** [review]
This is looking good. Just made a few comments for now. When this is in a final state, let me know and I will review + merge when complete.

File: docs/source/getting-started/index.rst (hunk 1 above)
Comment (johnmhoran) [reply docs/source/getting-started/index.rst]: Thanks @majurg -- I'll assign to myself and try to wrap up today or tomorrow.

File: docs/source/how-to-guides/load-your-data/open-save-sqlite-file.rst (hunk 2 above)
Comment (johnmhoran) [reply docs/source/how-to-guides/load-your-data/open-save-sqlite-file.rst]: fixed

File: docs/source/license.rst (hunk 3 above)
Comment (johnmhoran) [reply docs/source/license.rst]: Ticket entered -- we have a few open questions and then I'll take care of this asap.

**Review by steven-esser — APPROVED** [review]

**Pull request merged**